driver/src/price_finding/error.rs

```diff
@@ -36,6 +36,12 @@ impl From<&str> for PriceFindingError {
         PriceFindingError::new(error, ErrorKind::Unknown)
     }
 }
+
+impl From<(u16, u128)> for PriceFindingError {
+    fn from(_x: (u16, u128)) -> Self {
+        PriceFindingError::new("Some shit.", ErrorKind::Unknown)
+    }
+}
```

Comment:
nit: missing new-line between `impl` blocks.

Reply:
Actually this isn't supposed to be included! Whoops!